### PR TITLE
Feat/video timestamp

### PR DESCRIPTION
**Pull request by isaackps**
# In this PR:
- added quick video links dropdown
Screenshot 2021-01-19 at 3 04 32 PM

### CODE REVIEW

File: src/components/UI/ResourcesCard/ResourcesCard.stories.tsx [hunk 1]
```diff
@@ -17,6 +17,16 @@ const youtubeResource = {
       path: "/static/images/webinar/tradetrust-tech-webinar-1-demo.pdf",
     },
   ],
+  videoTimeStamps: [
```
Comment (simboonlong) [inline src/components/UI/ResourcesCard/ResourcesCard.stories.tsx]:
```suggestion
  videoChapters: [
```

File: src/components/UI/ResourcesCard/ResourcesCard.test.tsx [hunk 2]
```diff
@@ -18,6 +18,16 @@ const youtubeResource = {
       path: "/static/images/webinar/tradetrust-tech-webinar-1-demo.pdf",
     },
   ],
+  videoTimeStamps: [
+    {
```
Comment (simboonlong) [inline src/components/UI/ResourcesCard/ResourcesCard.test.tsx]: same

File: src/components/UI/ResourcesCard/ResourcesCard.tsx [hunk 3]
```diff
@@ -19,6 +20,10 @@ interface ResourcesCardProps {
       fileName: string;
       path: string;
     }[];
+    videoTimeStamps?: {
```
Comment (simboonlong) [inline src/components/UI/ResourcesCard/ResourcesCard.tsx]: same

File: src/components/UI/ResourcesCard/ResourcesCard.tsx [hunk 4]
```diff
@@ -34,19 +39,27 @@ export const ResourcesCard: FunctionComponent<ResourcesCardProps> = ({ details }
     watchLink,
     eventLink,
     eventSlides,
+    videoTimeStamps,
   } = details;
   const hasMedia = placeholderText || youtubeEmbedCode;
+  const [currentTimeStamp, setCurrentTimeStamp] = useState(0);
+
+  const convertSecondsToMinAndSec = (seconds: number): string => {
+    return `${~~(seconds / 60)}:${seconds % 60}m`;
```
Comment (simboonlong) [inline src/components/UI/ResourcesCard/ResourcesCard.tsx]: nice, din noe about double tilde ~~, also maybe wanna a unit test for this `convertSecondsToMinAndSec`?

File: src/components/UI/ResourcesCard/ResourcesCard.stories.tsx (hunk 1 above)
Comment (isaackps) [reply src/components/UI/ResourcesCard/ResourcesCard.stories.tsx]: ok

File: src/components/UI/ResourcesCard/ResourcesCard.test.tsx (hunk 2 above)
Comment (isaackps) [reply src/components/UI/ResourcesCard/ResourcesCard.test.tsx]: ok

File: src/components/UI/ResourcesCard/ResourcesCard.tsx (hunk 3 above)
Comment (isaackps) [reply src/components/UI/ResourcesCard/ResourcesCard.tsx]: ok

File: src/components/UI/ResourcesCard/ResourcesCard.tsx [hunk 5]
```diff
@@ -86,6 +99,25 @@ export const ResourcesCard: FunctionComponent<ResourcesCardProps> = ({ details }
...
+            className="rounded border border-grey-300 text-grey-700 p-2 mb-2"
+          >
+            {videoTimeStamps.map((eachTimeStamp, i) => {
+              return (
+                <DropdownItem
+                  key={i}
+                  data-testid="videoTimeStampsDropdown"
+                  onClick={() => setCurrentTimeStamp(eachTimeStamp.timeStamp)}
+                >
+                  {eachTimeStamp.title} [{convertSecondsToMinAndSec(eachTimeStamp.timeStamp)}]
```
Comment (simboonlong) [inline src/components/UI/ResourcesCard/ResourcesCard.tsx]: same, so it will be `videoChapter.timeStamp`

Comment (isaackps) [reply src/components/UI/ResourcesCard/ResourcesCard.tsx]: ok

File: src/components/UI/ResourcesCard/ResourcesCard.tsx (hunk 4 above)
Comment (simboonlong) [reply src/components/UI/ResourcesCard/ResourcesCard.tsx]: can be considered a utility, maybe can move to utils too i think

Comment (isaackps) [reply src/components/UI/ResourcesCard/ResourcesCard.tsx]: ok

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**